src/news/2014-03-24-floe-presentations-at-csun.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Floe Presentations at CSUN
+date: '2014-03-24'
+---
+Last week, the Floe team presented two papers at CSUN's annual
+[Internation Technology &amp; Persons With Disabilities conference](http://www.csun.edu/cod/conference/2014/sessions/index.php/public/website_pages/view/1)
+: "Fluid Infusion: Building the Next Gen
```

Comment:
Please fix the colon so there's no space before it. If the linter is complaining about the line length, you can add an exception for this case